src/bench/gcs_filter.cpp

```diff
@@ -67,7 +67,7 @@ static void GCSFilterDecodeSkipCheck(benchmark::Bench& bench)
     auto encoded = filter.GetEncoded();
 
     bench.run([&] {
-        GCSFilter filter({0, 0, BASIC_FILTER_P, BASIC_FILTER_M}, encoded, /*skip_decode_check=*/true);
```

Comment:
I think the entire `GCSFilterDecodeSkipCheck()` test can be removed? It's identical to `GCSFilterDecode()`

Reply:
Indeed, thanks for noticing :+1:

Reply:
Good catch. I'll remove it